composer.json

```diff
@@ -11,18 +11,22 @@
   ],
   "minimum-stability": "stable",
   "require": {
-    "php": ">=7.0",
+    "php": "^7.0 || ^8.1",
```

Comment:
Why not 8.0?

Reply:
@forevermatt What do you prefer?
(`8.1` matches our docker image.)

Reply:
Yeah, that's a good point. Using `"^7.0 || ^8.0"` would probably be better, because we have no reason to exclude use of this library by applications running PHP 8.0.